Okta.Xamarin/Okta.Xamarin.UITest.Shared/TestOktaConfig.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Okta.Xamarin.UITest
+{
+    // TODO: move OktaConfig to Okta.Xamarin.Shared to eliminate the need for this class
+    public class TestOktaConfig : IOktaConfig
+    {
+        public string ClientId { get; set; }
+        public string AuthorizeUri { get; set; }
+        public string RedirectUri { get; set; }
+        public string PostLogoutRedirectUri { get; set; }
+        public string Scope { get; set; }
+
+        public IReadOnlyList<string> Scopes
+        {
+            get; set;
+        }
+
+        public string OktaDomain { get; set; }
+        public string AuthorizationServerId { get; set; }
+        public TimeSpan ClockSkew { get; set; }
+
+        // TODO: refactor IOktaConfig and related to consolidate implementation
+        /// <summary>
+        /// Gets the Authorize Url used for logging in, which is either the <see cref="AuthorizeUri"/> if specified, or constructed from the <see cref="OktaDomain"/> and <see cref="AuthorizationServerId"/>.
+        /// </summary>
+        /// <returns>The computed Authorize Url used for logging in</returns>
+        public string GetAuthorizeUrl()
+        {
+            if (string.IsNullOrEmpty(AuthorizeUri))
+            {
+                return $"{OktaDomain}/oauth2/{AuthorizationServerId}/v1/authorize";
+            }
+            else
+            {
+                return AuthorizeUri;
+            }
+        }
+
+        /// <summary>
+        /// Gets the Access Token Url used for retrieving a token, which is constructed from the <see cref="OktaDomain"/> and <see cref="AuthorizationServerId"/>.
+        /// </summary>
+        /// <returns>The computed Access Token Url used for retrieving a token</returns>
+        public string GetAccessTokenUrl()
+        {
+            return $"{OktaDomain}/oauth2/{AuthorizationServerId}/v1/token";
+        }
+
+		public string GetLogoutUrl()
+		{
+			return $"{OktaDomain}/oauth2/{AuthorizationServerId}/v1/logout";
```

Comment:
Have you tested logout using the Okta org (AuthorizationServerId is "" in that case)? IIRC the URL is different. Take a look at this: https://github.com/okta/okta-aspnet/blob/5a2f459934e004a8e1211928eb548e1c1be3f378/Okta.AspNet.Abstractions/UrlHelper.cs#L14

Reply:
It actually seems to be a legacy issue, all other URLs (token, authorize) don't take the Okta Org Authz server into account. You might want to revisit this in the future.

Reply:
I'll plan to add a test for that.